pkg/testutils/lint/lint_test.go

```diff
@@ -1850,17 +1869,6 @@ func TestLint(t *testing.T) {
 		grepBuf.WriteString(")$")
 
 		filter := stream.FilterFunc(func(arg stream.Arg) error {
-			pkgPath := filepath.Join(cockroachDB, pkgScope)
-			pkgs, err := packages.Load(
-				&packages.Config{
-					Mode: packages.NeedImports | packages.NeedName,
-					Dir:  crdbDir,
-				},
-				pkgPath,
-			)
-			if err != nil {
-				return errors.Wrapf(err, "error loading package %s", pkgPath)
-			}
```

Comment:
The check for `len(pkgs) == 0` below is redundant, since it was lifted above.

Reply:
whoops, missed that one, nice catch.